### ISSUE TITLE
amp-lightbox-gallery: issue when only 2 images

### ISSUE THREAD

**Issue by javierparis** [ampproject/amphtml#22608]
When triggering an amp-lightbox-gallery that **only has 2 images**, it's not possible to swipe to the right, and swiping to the left has a flickering effect and it's not consistently displaying the image

How to reproduce it:
Go to the playground for amp-lightbox-gallery
https://playground.amp.dev/?url=https%3A%2F%2Fpreview.amp.dev%2Fdocumentation%2Fexamples%2Fcomponents%2Famp-lightbox-gallery&mode=Galaxy+S5
and leave only 2 images, something like this:
```
<div>
    <amp-img lightbox src="https://preview.amp.dev/static/samples/img/road-1.jpg" width="300" height="200" layout="responsive"></amp-img>
    <p class="paragraph">Lorem ....</p>
    <amp-img lightbox src="https://preview.amp.dev/static/samples/img/road-2.jpg" width="300" height="200" layout="responsive"></amp-img>
    <p class="paragraph">Lorem ....</p>
  </div>
```
Reproduce it using Google Chrome tools, trigger the device toolbar, and choose iphone 6/7/8 for instance. Arrows seem to work though

**Comment by sparhami** [comment]
I think we should wait for the switch to use `<amp-base-carousel>` in lightbox (#21568) for this one. I tried this by enabling the `'amp-base-carousel-lightbox` and `layers` experiments and it works correctly. Currently, it tries to loop for two slides, which can be weird, but the change to disable looping for two slides should go live soon.

Here is what we still need to launch this: https://github.com/ampproject/amphtml/issues/21568#issuecomment-497802999

**Comment by maciejmackowiak** [comment]
@sparhami 
Hi,
Any updates about this one?
I've got the exactly same issue.

Thanks
Maciej

**Comment by archon810** [comment]
Have a look at this report please https://github.com/android-police/androidpolice-public/issues/100.

**Comment by sparhami** [comment]
We have a new version of amp-carousel which will fix this, but haven't changed the lightbox gallery over to use it yet. We plan on changing over in the next few weeks.

**Comment by maciejmackowiak** [comment]
That is a great news, thanks for quick response.

**Comment by westonruter** [comment]
> We have a new version of amp-carousel which will fix this, but haven't changed the lightbox gallery over to use it yet. We plan on changing over in the next few weeks.

@sparhami By new version, do you mean `amp-carousel v0.2 as opposed to v0.1? If so, then bumping the version in the AMP plugin (https://github.com/ampproject/amp-wp/pull/3115) should fix this, correct?

**Comment by archon810** [comment]
Any updates here?

**Comment by archon810** [comment]
Hi, asking again about the fix for this issue.

**Comment by nainar** [comment]
This is fixed if you are using carousel v 0.2

**Comment by ZdejPoham** [comment]
So, if I have 2 pictures on a page, that cannot be put inside an amp-crousel v0.2, I'm stuck with this issue. Brilliant!
